app/(tabs)/summary/main-summary.tsx

```
@@ -86,72 +111,106 @@ const Summary = () => {
+            return (
+              <View style={{justifyContent: 'center', alignItems: 'center'}}>
+                <Text
+                  style={{fontSize: 12, color: 'black', fontWeight: 'bold'}}>
+                  {formatPrice(_.sumBy(data, 'value'))}
+                </Text>
+                {data.slice(0, 4).map(({label, value}) => (
+                  <Text
+                    style={{
+                      fontSize: 10,
+                      color: 'black',
+                    }}>{`${shortenText(label)}(${formatPrice(value)})`}</Text>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)